### PR TITLE
Drop the Player.timezone column since we have tz (string) now

### PR DESCRIPTION
**Pull request by alalonde**
As suggested by @pakokrew [here](https://github.com/MetaFam/TheGame/pull/230#pullrequestreview-554023859)

### CODE REVIEW

File: hasura/metadata/tables.yaml [hunk 1]
```diff
@@ -166,54 +166,6 @@
...
-      - role
-      - timezone
-      - tz
-      - username
-      filter:
-        id:
-          _eq: X-Hasura-User-Id
-      check: {}
-      set:
-        id: x-hasura-User-Id
```
Comment (hammadj) [inline hasura/metadata/tables.yaml:216]: What are all these permission updates? Doesn't seem related to the timezone column

Comment (alalonde) [reply hasura/metadata/tables.yaml:216]: Ugh.. there must be a bug in Hasura or something, as these permissions keep getting dropped when I make schema changes.
